### ISSUE TITLE
Path of the road slightly changes after creating a crossing

### ISSUE THREAD

**Issue by boformer** [bernardd/Crossings#1]
This is a possible problem because the zones get re-arranged (and existing buildings disappear).

This only affects curved roads.

Nice work so far!

**Comment by bernardd** [comment]
Man, there's like a billion problems with it at the moment (though about half a billion less than earlier tonight).
How did you even find this? I haven't released it anywhere :)

**Comment by bernardd** [comment]
Also, in regard to the specific issue, that's one problem I may not be able to solve. There's no sane way (that I can figure out) to add crossings without adding a node, and there's no way to do that without potentially slightly tweaking the alignment of roads.
Though thinking about it, I _may_ be able to get it to at least highlight anything that's going to be destroyed, much like what happens now for road construction. I'll make a note to look at that.

**Comment by boformer** [comment]
> How did you even find this? I haven't released it anywhere :)

I compiled it myself. I'm watching your repo ;)

**Comment by bernardd** [comment]
Yeah, I saw that, I was just wondering how you stumbled across my repo in the first place :)
Regardless, it's now on the workshop, so you can grab the "official" version if you're so inclined: http://steamcommunity.com/sharedfiles/filedetails/?id=427258853&tscn=1429358013
Thanks for taking an interest :)
